homeassistant/components/lupusec/__init__.py

```diff
@@ -39,44 +41,77 @@
     extra=vol.ALLOW_EXTRA,
 )
 
-LUPUSEC_PLATFORMS = [
+LUPUSEC_PLATFORMS: list[Platform] = [
```

Comment:
Please rename this to PLATFORMS while you are at it

Reply:
Done.